### PR TITLE
Update the requires_ansible version

### PR DESCRIPTION
**Pull request by Ruchip16**
##### SUMMARY

nightly tests are resulting in failure due to the older ansible version used, this PR updates the `requires_ansible` version to immediate next supported version.

##### ISSUE TYPE

- CI Pull Request

##### COMPONENT NAME

meta/runtime.yaml

### CODE REVIEW

File: changelogs/fragments/v10.0.0_bump_ansible_core.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
```
Comment (NilashishC) [inline changelogs/fragments/v10.0.0_bump_ansible_core.yaml:2]: Can we use the same wording that we did before - https://github.com/ansible-collections/cisco.iosxr/pull/461/files#diff-badba4dd47cd1a24e2765eb5c5f420a529872e06a0ce09251c35ea2cb5b4ebd9R691-R699? Except for the "last known version compatible with...." part.

File: changelogs/fragments/v10.0.0_bump_ansible_core.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (NilashishC) [inline changelogs/fragments/v10.0.0_bump_ansible_core.yaml:2]: You missed the release_summary section.

```
release_summary: Starting from this release, the minimum `ansible-core` version this collection requires is `2.15.0`.
```

Comment (NilashishC) [reply changelogs/fragments/v10.0.0_bump_ansible_core.yaml:2]: Well, now that I think of it, we can also add: "The last known version compatible with ansible-core<2.15 is `v9.0.0`.

Comment (Ruchip16) [reply changelogs/fragments/v10.0.0_bump_ansible_core.yaml:2]: yes yes i am updating that, i was merging the pre-commit PR and then renaming this to Major release PR thats why kept the changes on hold.

**Pull request closed without merging**